aries-site/src/pages/learn/how-to-add-search-and-filter-to-datatable-with-data.mdx

```diff
@@ -0,0 +1,196 @@
+```
+
+Next, add `view` and `onView` as properties on Data. When the filter change, `onView` will provide the new values. Use this to update the `view` state variable and reset the page to 1.
+
+```
+<Data
+  data={data}
+  view={view}
+  onView={nextView => {
+    setView(nextView);
+    setPage(1);
+  }}
+  total={numberItems}
+  toolbar
+>
+```
+
+To fetch the data again when the value of `view` changes, pass `view` as a parameter in `fetchData` and include `view` as an item in the dependency array of the useEffect.
+
+```
+useEffect(() => {
+  const fetchData = fetchPayloads;
+  fetchData(page, view)
+    .then((d) => setData(d))
+    .catch((error) => console.log(`Unable to fetch data: ${error}`));
+}, [page, view]);
+```
+
+At this point, your project should look like this:
+
+<Box
+  width="medium"
+  round="xsmall"
+  overflow="auto"
+  margin={{ bottom: 'medium' }}
+>
+  <Image
+    src="/learnImages/data-simple-toolbar.png"
+    alt="Toolbar containing a search input and filter button located above a DataTable displaying SpaceX launch data."
+    fit="contain"
+  />
+</Box>
+
+<Notification
+  status="learn"
+  width="large"
+  margin={{ bottom: 'medium' }}
+  message='If you type something into the search input and press "Enter", the data in the DataTable will be filtered to those that match the search term.'
+/>
+
+<Notification
+  status="learn"
+  width="large"
+  message="If you click on the filter button, Data has populated the filters to include all of the data attributes. The input type (Select, CheckBoxGroup, etc.) is determined based on the type of data and amount of results for that attribute."
+/>
+
+## Customize filtered attributes
+
+To customize which attributes are filterable, use `properties` property on Data. In this case, let's only filter by "type" and "mass_lbs".
+
+For the ease of this guide, we've hardcoded the options for "type" and "mass_lbs", but handling this dynamically is recommended for applications in production.
+
+Add the following to Data:
+
+```
+<Data
+  data={data}
+  properties={{
+    type: {
+      label: 'Type',
+      options: [
+        'Dragon 1.0',
+        'Dragon 1.1',
+        'Dragon Boilerplate',
+        'Satellite',
+      ],
+    },
+    mass_lbs: {
+      label: 'Mass',
+      range: { min: 0, max: 1400 },
+    },
+  }}
+  view={view}
+  onView={nextView => {
+    setView(nextView);
+    setPage(1);
+  }}
+  total={numberItems}
+  toolbar
+>
+    <DataTable columns={columns} />
+</Data>
+```
+
+Now, when the filters menu is opened, the filters will look like this:
+
+<Box
+  width="medium"
+  round="xsmall"
+  overflow="auto"
+  margin={{ bottom: 'medium' }}
+>
+  <Image
+    src="/learnImages/data-simple-filters.png"
+    alt="Filters for type and mass displayed in a drop over a datatable presenting SpaceX data."
+    fit="contain"
+  />
+</Box>
+
+Congratulations! You’ve used Data to add search and filter controls to manage what data should be rendered in a DataTable.
```

Comment:
wonder if we should also use `notification` here to have this stand out a little more that they completed the task

Reply:
That's an option. I can try that.

Reply:
I decided to leave as if for now -- the smaller text size of the notification felt awkward for a "completed" step.